entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -l
+set -eo pipefail
+
+cp -r /grammar ./wgsl/
+export PATH="$(python3 -m site --user-base)/bin:${PATH}"
+git init
```

Comment:
Why is this making a git repo?

Reply:
This helps avoid an issue where container action only recognizes when fetch depth is larger than 1. Instead of running longer fetches, doing this no harm operation fixes, but noted it with a comment!

Reply:
(I mean comment in script)

Reply:
(It just reinitializes than make a new one)